scripts/dev/update_version.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2018 Andy Mender <andymenderunix@gmail.com>
+
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+from datetime import date
+import subprocess
+
+from lxml import etree
+
+# TODO: move to global constants?
+appdata_path = "misc/qutebrowser.appdata.xml"
+version_xpath = '//*[@type="desktop"]/releases'
+
+
+def bump_version(version_leap = "patch"):
+    """Update qutebrowser release version in .bumpversion.cfg
+
+    :param version_leap: define the jump between versions ("major", "minor", "patch")
+    """
+
+    # make sure the correct version jump was chosen
+    if version_leap not in ("major", "minor", "patch"):
+        raise ValueError("version_leap takes values of 'major', 'minor' or 'patch' only.")
+
+    # NOTE: this may rely on the python version used to launch the function
+    # and whether the wrapper script 'bumpversion' was provided
+    subprocess.run(['bumpversion', version_leap, '--allow-dirty'])
```

Comment:
Please use `check=True` so the script aborts when this failed.

Reply:
Probably `sys.executable, '-m', 'bumpversion'` or similar would work better, as it doesn't rely on it being in the `$PATH`.

Reply:
Not sure if it makes sense to specify `--allow-dirty` by default - or is that needed because of stuff like the `.tox` folder?